CHANGELOG.rst

```diff
@@ -1,3 +1,28 @@
+**JDBC Driver 3.13.31**
+
+- \ Fetch metadata.getTableTypes() from the backend service" (#1392)
```

Comment:
are we going to put commit numbers in the CHANGELOG?

Reply:
we will be removing this looks like automation added this commits